website/docs/feature-flag-tutorials/use-cases/security-compliance.md

```diff
@@ -0,0 +1,240 @@
+
+![The diagram showing the relationship between root roles and project roles in Unleash.](/img/root-and-project-roles.png)
+
+:::note
+
+One of the key responsibilities of the _admin role_ is assigning users as project owners and members.
+_Editor roles_ can be designated as project owners to manage projects.
+_Viewer roles_ can be designated as project members to observe projects. If viewers are not assigned a project member role, they can only observe the actions taken in a project.
+
+:::
+
+To create your _new roles_, create [custom root roles](/how-to/how-to-create-and-assign-custom-root-roles) and [custom project roles](/how-to/how-to-create-and-assign-custom-project-roles), where you can define the privileges and limitations beyond the predefined roles we have built into Unleash. For more recommendations on setting up permissions for users, read our guide on [using feature flags at scale](/topics/feature-flags/best-practices-using-feature-flags-at-scale).
+
+Unleash can handle all of the complexities that come with access controls and we make that process simple for you. Set up roles and permissions so you’re not only in complete control of how your feature flag system is administered, you’re adhering to compliance standards.
+
+Next, we’ll explore how to extend access controls in Unleash for network security.
+
+## Set up access controls for network security
+
+Securing your network layer is non-negotiable for building and using software systems. It’s foundational knowledge and good practice for any organization with engineering teams deploying software. Now that you understand how to set up and manage users at scale in Unleash, you can begin configuring Unleash in your applications and services [using our SDKs](/reference/sdks#official-sdks). Using the admin UI is an easy, direct way to make changes to your feature flags, projects, rollout strategies, and more. But when you’re making calls to our frontend API from your services, you can update cross-origin resource sharing (CORS) settings to control application access to Unleash. You can also set up IP allow lists to restrict access to Unleash.
+
+### Set up CORS policies for Frontend API security
+
+It was noted in [the previous section](#configure-role-based-access-control-for-administrators-and-developers) that access controls are about security measures in place at every entry point to your systems. This means they go beyond just user management. Extend these controls to [Frontend APIs](/reference/api/unleash/frontend-api) to prevent security risks across your stack.
+
+From an architectural perspective, we implement this by allowing Unleash users to define cross-origin resource sharing (CORS) policies to prevent unwarranted access to your feature flag system. From a compliance perspective, this demonstrates that your organization leverages a feature flag system that takes a multi-faceted approach to security.
```

Comment:
Comment from Ivar:

in reality: This is a requirement from modern browser. They will not connect to another domain if it is not explicitly allow cross-origin access from the provided domain

Suggestion to change to something like:
Modern browsers enforce cross-origin resource sharing (CORS) restrictions, preventing connections to other domains unless explicitly allowed. In Unleash, you can define cross-origin resource sharing (CORS) policies to prevent unwarranted access to your feature flag system. This not only ensures compliance with browser security requirements but also reinforces a multi-layered security approach within your organization.

Reply:
Updated, thanks!